draft-ietf-v6ops-464xlaton.xml

```diff
@@ -290,68 +270,82 @@ Therefore, as long as the host is not using DS-Lite, it MAY use 192.0.0.0/29 for
 <name>
 CLAT IPv6 Addresses
 </name>
+<section anchor="slaac">
+<name>Obtaining CLAT IPv6 Addresses</name>
 <t>
 Section 6.3 of <xref target="RFC6877"/> recommends that the CLAT instance acquires a dedicated /64 for translating between IPv4 and IPv6, and only uses a single interface IPv6 address if a dedicated prefix is not available via DHCPv6-PD.
 However, deployments where each node can obtain a dedicated /64 just for CLAT are rather uncommon, especially in environments like enterprise networks, Wi-Fi hotspots, etc. 
 Quite often the CLAT instance uses a single IPv6 address as a source for all IPv4 traffic translated by CLAT. 
-In particular, in a single address model (see <xref target="v4addr"/>) the CLAT instance only need a single CLAT IPv6 address, so obtaining a /64 is wasteful.
+In particular, if the CLAT only provides the IPv4 connectivity to applications local to the node (or if the node can perform stateful NAT44) the CLAT instance only need a single CLAT IPv6 address, so obtaining a /64 is wasteful.
 For instance, a home network that gets a /60 from its ISP can only connect up to 15 CLAT-enabled devices before it runs out of available prefixes.
-Even in a dedicated prefix model, the CLAT instance can first perform stateful NAT44 to translate all IPv4 addresses from the dedicated prefix to a single IPv4 address, and then perform stateless CLAT. 
+Even if the node extends IPv4 connectivity downstream, the CLAT instance can first perform stateful NAT44 to translate all IPv4 addresses used by downstream devices to a single IPv4 address, and then perform stateless CLAT. 
 </t>
 <t>
-This document updates <xref target="RFC6877"/> by removing the requirement to acquire a dedicated /64 prefix for the purpose of sending and receiving statelessly translated packets.
+This document updates <xref target="RFC6877"/> by relaxing the requirement to acquire a dedicated /64 prefix for the purpose of sending and receiving statelessly translated packets.
 The following recommendations are made instead:
 </t>
 <ul>
 <li>
 <t>
-In a single-address model, the CLAT instance SHOULD NOT obtain a dedicated /64 for the purpose of sending and receiving statelessly translated packets.
+If the node is extending IPv4 connectivity downstream and is not performing stateful NAT44 (so it needs to translate downstream IPv4 addresses to IPv6 statelessly), the node 
+SHOULD obtain a dedicated /64 prefix (e.g. via DHCPv6-PD).
```

Comment:
```suggestion
If the node is extending IPv4 connectivity downstream, the node 
SHOULD obtain a dedicated /64 prefix (e.g. via DHCPv6-PD) for stateless translation of downstream IPv4 addresses unless it chooses to perform stateful NAT44 from the downstream IPv4 addresses to a single IPv4 address for stateless translation.
```

Again making stateless CLAT the first class citizen, which is the original definition of CLAT in 6877 (using NAT44 is the newly introduced weirdness).